src/app/pages/field-trip-summary/field-trip-summary.page.ts

```diff
@@ -95,6 +95,13 @@ export class FieldTripSummaryPage implements AfterViewInit {
 		}
 
 		this.getDateAndDuration();
+
+		if ((await Network.getStatus()).connected){
```

Comment:
Ikke noe poeng i å gjøre hele ionViewWillEnter async for å kjøre Network.getStatus(). Kjør heller:
`Network.getStatus().then((status) => {
    this.connectedToNetwork = status.connected.
})`